models/stg_iterable__campaign_metrics.sql

```diff
@@ -0,0 +1,86 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_iterable__campaign_metrics_tmp') }}
+    where not coalesce(_fivetran_deleted, true)
+
+),
+
+fields as (
+
+    select
+        /*
+        The below macro is used to generate the correct SQL for package staging models. It takes a list of columns 
+        that are expected/needed (staging_columns from dbt_iterable_source/models/tmp/) and compares it with columns 
+        in the source (source_columns from dbt_iterable_source/macros/).
+        For more information refer to our dbt_fivetran_utils documentation (https://github.com/fivetran/dbt_fivetran_utils.git).
+        */
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_iterable__campaign_metrics_tmp')),
+                staging_columns=get_campaign_metrics_columns()
+            )
+        }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+        id as campaign_id,
+        date,
```

Comment:
might wanna rename from `date` as i could see it causing issues

Reply:
I agree